src/api/admin/admin.test.js

```diff
@@ -3,9 +3,11 @@ const request = require('supertest');
 const app = require('../../app');
 
 describe('GET /admin', () => {
-  it('Should respond with a 401 status code', () =>
+  it('Should respond with a 401 status code', done => {
     request(app)
       .get('/admin')
       .expect('Content-Type', /json/)
-      .expect(401));
+      .expect(401)
+      .end(done);
```

Comment:
Done can be passed inside the expect
`.expect(401, done);`

Reply:
Thanks for that tip, Ill make that change and make sure it is consistent on all tests.

Reply:
Done